android/src/main/res/values/strings.xml

```diff
@@ -263,4 +263,6 @@
     <string name="notifications_delivered_when_user_interaction_is_required_to_establish_the_vpn_tunnel">Notifications delivered when user interaction is required to establish the VPN tunnel.</string>
     <string name="optional_notifications_which_display_the_status_of_the_vpn_tunnel">Optional notifications which display the status of the VPN tunnel.</string>
     <string name="notifications_delivered_when_a_file_is_received_using_taildrop">Notifications delivered when a file is received using Taildrop.</string>
+    <string name="health_channel_name">Health</string>
+    <string name="health_channel_description">Notifications about the health of the VPN connection.</string>
```

Comment:
I'm not convinced we want to expose this to the user as "Health"...  It has a slightly negative context.  @sonovawolf should probably weigh in here...

Reply:
Changed this for now to be a bit more generic:

> **Errors and warnings**
> This notification category is used to deliver important status notifications and should be left enabled. For instance, it is used to notify you about errors or warnings that affect Internet connectivity.

Reply:
Will review tomorrow with @sonovawolf